.devcontainer/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # [Choice] Ubuntu version (use ubuntu-22.04 or ubuntu-18.04 on local arm64/Apple Silicon): ubuntu-22.04, ubuntu-20.04, ubuntu-18.04
 ARG VARIANT=ubuntu-24.04
-FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+FROM mcr.microsoft.com/vscode/devcontainers/base:debian
 
 # Install additional OS packages.
 RUN apt-get update -qq && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
Let's also install `graphviz`. I noticed the unit tests fail in the dev container.